app/components/dashboard/run/left-panel/component.js

```diff
@@ -165,5 +166,23 @@ export default Component.extend(FullScreenMixin, {
         denyDelete() {
             return true;
         },
+        
+        exportZip(id) {
+          const token = this.get('tokenHandler').getWholeTaleAuthToken();
+          let url = `${config.apiUrl}/tale/${id}/export`;
+          window.open(url + '?token=' + token + '&taleFormat=native', '_newtab');
+        },
+
+        exportBag(id) {
```

Comment:
Could you collapse that into one function by passing `taleFormat` as 2nd argument?